pkg/ocm/accessrequests.go

```diff
@@ -20,3 +24,15 @@ func (c *Client) CreateDecision(accessRequest string, decision string, justifica
 	}
 	return nil
 }
+
+func (c *Client) GetAccessRequest(id string) (*v1.AccessRequest, bool, error) {
```

Comment:
Is there ever a situation where this function could return "true" for the second param and an error in the 3rd param? I ask because it's unclear to me what the second parameter's purpose is.

Reply:
it's impossible. Our code guarantees the error is always nil when 2nd return is true.
The purpose of the 2nd return is to clarify notfound error and other errors. So that we can give more user-friendly message in the output when cannot find access request using the specified id

Reply:
I'm wondering if `GetAccessRequest` should just return an error for a 404 and let it bubble up to be handled further up, or maybe even just a nil `*v1.AccessRequest`. That way we can avoid having to check multiple return params from call sites.